ribbon/constants/addresses.json

```diff
@@ -0,0 +1,17 @@
+{
+  "kovan": {
+    "swap": {
+      "address": "0x58848824baEb9678847aF487CB02EAba782FECB5"
```

Comment:
I'm thinking what's the best way to have these.

We don't want to have it hardcoded, but more to be a configuration for the SDK,
or for the part of the SDK that uses them (only the verify wallet?).

Reply:
Ok so looking below the "configuration" object we need, 
is for creating a connection to a Contract.

Reply:
An example draft is in https://github.com/pdonorio/sdks/blob/paolo/config-setup-proposal/ribbon/classes.py#L53-L112

which allows to
<img width="497" alt="Schermata 2022-04-08 alle 13 40 20" src="https://user-images.githubusercontent.com/9340175/162428675-557a976c-ecba-4a8c-83e9-74a9f18332aa.png">

Reply:
Updated: 
- the [ContractConfig](https://github.com/pdonorio/sdks/blob/paolo/config-setup-proposal/ribbon/classes.py#L87-L103) definition
- the use of this config [in the Contract abstraction](https://github.com/pdonorio/sdks/blob/paolo/config-setup-proposal/ribbon/contract.py#L45-L61)
- removed the specific init from the SwapContract ([ref](https://github.com/pdonorio/sdks/blob/paolo/config-setup-proposal/ribbon/swap.py#L24))

This can work like this:
<img width="574" alt="Schermata 2022-04-08 alle 15 33 47" src="https://user-images.githubusercontent.com/9340175/162446356-031ef696-4652-4eeb-9ebe-4f0d3dc3ae92.png">

The idea is to be able to create a set of contract configs that we will be keeping
to be used based on the RFQ.

Reply:
https://github.com/stevenwal/sdks/pull/1

Reply:
Solved